tests/unit/views/pacbio/PacbioRun.spec.js

```diff
@@ -201,4 +208,30 @@ describe('Run.vue', () => {
         })
     })
 
+    describe('#reset', () => {
+
+        beforeEach(() => {
+            wrapper = shallowMount(PacbioRun, {
+            store, 
+            router,
+            localVue,
+            propsData: { id: 1},
+            methods: {
+               provider() { return }
+            }
+        })
+        pacbioRun = wrapper.vm
+
+            pacbioRun.showAlert = jest.fn()
+            pacbioRun.newRun = jest.fn()
+            pacbioRun.redirectToRuns = jest.fn()
+        })
+
```

Comment:
I don't think you need to mount a new component to test this. Also, you are passing in here `propsData: { id: 1 }`, which implies it is an existing run, not a new run. So that might suggest a bug is the code or a test not quite right.